### DISCUSSION TITLE
Is the MVC architecture needed for Lona programs?

### DISCUSSION THREAD

**Discussion by metaperl** [lona-web-org/lona#492] [General]
Classical UI development followed the model-view-controller pattern as shown at this site - https://puremvc.org/

is the creation of discrete model, view and controller useful for larger Loan programs?

**Comment by fscherf** [comment]
Hi @metaperl!
Lona is very object-oriented because this pattern fits the problem Lona solves very well, but it does not implement mvc itself, and you are not required to adapt mvc in your projects. I like to use Lona+Django for larger projects or if I need a database and authentication.
When Lona and Django are used together, Lona only implements the "View" part of MVC. Django does the rest.
